Fix build warnings on Python 2

```
jump/jump.cpp:16:12: warning: implicit conversion loses integer precision:
          'int64_t' (aka 'long long') to 'int32_t' (aka 'int') [-Wshorten-64-to-32]
    return b;
    ~~~~~~ ^
1 warning generated.
```

--- a/jump/jump.cpp
+++ b/jump/jump.cpp
@@ -13,5 +13,5 @@ JumpConsistentHash(uint64_t key, int32_t num_buckets)
         j = (b + 1) * (double(1LL << 31) / double((key >> 33) + 1));
     }
 
-    return b;
+    return (int32_t)b;
 }

